workspaces/resource-optimization/plugins/resource-optimization-common/README.md

```diff
@@ -1,5 +1,7 @@
-# @backstage-community/plugin-resource-optimization-common
+# Resource Optimization Common
 
-Welcome to the common package for the resource-optimization plugin!
+Shared isomorphic code for the resource-optimization plugin.
```

Comment:
@jkilzi do you think it will be good to specify here which functionality is shared and which plugin (frontend/backend) is consuming it? Also shall we mention how it is getting installed (as a dependency) or it is not needed? Let me know your thoughts around the same! 🙌🏻

Reply:
Those are implementation details, that might be of interest for developers instead of users of this plug-in.

Reply:
By the way, each package has a generated api-report.md file that brings more information about those details.